spec/policies/application_policy_spec.rb

```diff
@@ -0,0 +1,43 @@
+describe ApplicationPolicy do
+  let(:user_context) { instance_double(UserContext, :group_uuids => ["123-456"]) }
+  let(:record) { create(:portfolio) }
+  let(:subject) { described_class.new(user_context, record) }
+
+  it "#index?" do
+    expect(subject.index?).to be_falsey
+  end
+
+  it "#show?" do
+    expect(subject.show?).to be_falsey
+  end
+
+  it "#create?" do
+    expect(subject.show?).to be_falsey
+  end
+
+  it "#new?" do
+    expect(subject.show?).to be_falsey
+  end
+
+  it "#update?" do
+    expect(subject.show?).to be_falsey
+  end
+
+  it "#edit?" do
+    expect(subject.show?).to be_falsey
+  end
```

Comment:
All of these I think are just copy/paste errors, need to change out the `.show?` for the right method in each one :)